<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSketchOptions</class>
 <widget class="QWidget" name="WdgSketchOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>210</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Line width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KDoubleNumInput" name="lineWidthSPBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
        <property name="exponentRatio">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offset scale:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KDoubleNumInput" name="offsetSPBox">
        <property name="value">
         <double>30.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Density:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KDoubleNumInput" name="densitySPBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>406</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="distanceDensityCHBox">
        <property name="text">
         <string>Use distance density</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="simpleModeCHBox">
        <property name="text">
         <string>Simple mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="magnetifyCHBox">
        <property name="text">
         <string>Magnetify</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="connectionCHBox">
        <property name="text">
         <string>Paint connection line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="randomRGBCHbox">
        <property name="text">
         <string>Random RGB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="randomOpacityCHbox">
        <property name="text">
         <string>Random opacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="distanceOpacityCHbox">
        <property name="text">
         <string>Distance opacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
